spec/lib/coach/middleware_spec.rb

```diff
@@ -17,16 +17,16 @@
       before { middleware_class.provides(:foo, :bar) }
 
       it "returns true" do
-        expect(middleware_class.provides?(:foo)).to be_truthy
-        expect(middleware_class.provides?(:bar)).to be_truthy
+        expect(middleware_class).to be_provides(:foo)
```

Comment:
Urgh. This is weird. I wouldn't understand what this is doing by just looking at it.

Can we exclude this cop here? Or is it only me who struggles to parse this?

Reply:
Didn't spot this. Definitely!